src/olympia/devhub/views.py

```diff
@@ -1038,31 +1037,19 @@ def version_edit(request, addon_id, addon, version_id):
                     _log_max_version_change(addon, version, form.instance)
 
         if 'version_form' in data:
-            # VersionForm.save() clear the pending info request if the
-            # developer specifically asked for it, but we've got additional
-            # things to do here that depend on it.
-            had_pending_info_request = bool(addon.pending_info_request)
             data['version_form'].save()
 
             if 'approval_notes' in version_form.changed_data:
-                if had_pending_info_request:
-                    log_and_notify(amo.LOG.APPROVAL_NOTES_CHANGED, None,
```

Comment:
do we need to `log_and_notify` when there's a delayed rejection pending instead?

Reply:
I don't think so. Delayed rejections can't be cancelled by regular reviewers, a developer needs to submit a new version to fix the issues (or appeal to an admin). So notifying here doesn't help much. (Reviewers will be subscribed to notifications about new versions for that add-on when they submit a delayed rejection, though)